src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -131,10 +133,15 @@ const PageHeaderContainer = styled.div`
 `;
 
 interface Props {
-  scenarioId: string;
+  scenario: Scenario;
 }
 
-const FacilityInputForm: React.FC<Props> = ({ scenarioId }) => {
+const FacilityInputForm: React.FC<Props> = ({ scenario }) => {
+  const scenarioId = scenario.id;
+  const [syncDataModalFacility, setSyncDataModalFacility] = useState<
+    string | null
+  >(null);
+  const useReferenceFacilities = useFlag(["enableShadowData"]);
```

Comment:
there is a hook called `useReferenceFacilitiesEligible` that you probably want to use here instead; there are other criteria besides the feature flag that determine whether the feature should be available.